sources/api/prairiedog/src/bootconfig.rs

```diff
@@ -94,23 +95,14 @@ where
 
 /// Retrieves boot config related Bottlerocket settings. If they don't exist in the settings model,
 /// we return `None` instead.
-async fn get_boot_config_settings<P>(socket_path: P) -> Result<Option<BootSettings>>
+async fn get_boot_config_settings<P>(config_path: P) -> Result<Option<BootSettings>>
 where
     P: AsRef<Path>,
 {
-    let uri = "/settings";
-    let settings: serde_json::Value =
-        schnauzer::v1::get_json(socket_path, uri, Some(("prefix", "boot")))
-            .await
-            .context(error::RetrieveSettingsSnafu)?;
-
-    match settings.get("boot") {
-        None => Ok(None),
-        Some(boot_settings_val) => Ok(Some(
-            serde_json::from_value(boot_settings_val.to_owned())
-                .context(error::BootSettingsFromJsonValueSnafu)?,
-        )),
-    }
+    let config_str = fs::read_to_string(config_path.as_ref()).context(error::ReadFileSnafu {
+        path: config_path.as_ref().to_path_buf(),
+    })?;
+    toml::from_str(config_str.as_str()).context(error::InputTomlSnafu)
```

Comment:
If we're not using await, then the function probably shouldn't be async and we maybe shouldn't even use tokio. On the other hand, if we're using async, we should use `tokio::fs` and await.

Right now this sort of splits the difference and ends up making blocking I/O calls in async functions, which is something we need to be vigilant about.

Reply:
Most of the methods pre-existing in this tool are written with tokio async but without apiserver they technically don't seem to need to be, so the question really here is, would you like me to de-async this or just update my fs to use tokio::fs?

Reply:
De-asyncified prairiedog